app/hyperterm.js

```diff
@@ -269,30 +273,28 @@ export default class HyperTerm extends Component {
     }
   }
 
-  changeFontSize (value) {
-    const uid = this.state.sessions[this.state.active];
-    const term = this.refs[`term-${uid}`];
-    if (term) {
-      const size = term.term.prefs_.get('font-size');
-      term.term.prefs_.set('font-size', size + value);
-    }
+  changeFontSize (value, increment = false) {
+    this.setState({
+      fontSize: increment ? this.state.fontSize + value : value,
+      fontSizeIndicatorShowing: true
+    });
+
+    clearTimeout(this.fontSizeIndicatorTimeout);
+    this.fontSizeIndicatorTimeout = setTimeout(() => {
+      this.setState({ fontSizeIndicatorShowing: false });
+    }, 1500);
   }
 
   resetFontSize () {
-    const uid = this.state.sessions[this.state.active];
-    const term = this.refs[`term-${uid}`];
-    if (term) {
-      // TODO: once we have preferences, we need to read from it
-      term.term.prefs_.set('font-size', 12);
-    }
+    this.changeFontSize(12);
   }
 
   increaseFontSize () {
-    this.changeFontSize(1);
+    this.changeFontSize(1, true);
```

Comment:
`{ relative: true }` ?
Also, are we cleaning up `fontSizeIndicatorTimeout` on unmount?